core/src/test/java/org/elasticsearch/search/suggest/SuggestSearchIT.java

```diff
@@ -1108,6 +1119,231 @@ public void testSuggestWithManyCandidates() throws InterruptedException, Executi
         // assertThat(total, lessThan(1000L)); // Takes many seconds without fix - just for debugging
     }
 
+    @Override
+    protected Collection<Class<? extends Plugin>> nodePlugins() {
+        return Collections.singleton(DummyTemplatePlugin.class);
+    }
+
+    public static class DummyTemplatePlugin extends Plugin implements ScriptPlugin {
```

Comment:
Makes sense to me. I only point it out in case you hadn't noticed it and it could save you any code.